composer.json

```diff
@@ -46,8 +47,7 @@
     },
     "scripts": {
         "test": [
-            "vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh",
-            "SYMFONY_PHPUNIT_REMOVE=\"symfony/yaml\" vendor/bin/simple-phpunit"
+            "vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh"
```

Comment:
Why use simple phpunit? For me just makes development difficult. Is there still a reason to use it?